bridge/resources/resource.go

```diff
@@ -74,8 +74,43 @@ func (r *Resource) GetHref() string {
 	return r.Href
 }
 
+type SupportedOperation int
```

Comment:
@coderabbitai write unit tests for `SupportedOperation` type and for `func (r *Resource) SupportsOperations() SupportedOperation`